task_2/1.js

```diff
@@ -0,0 +1,12 @@
+// 1 - Реализовать функцию, которая принимает текст, и возвращает массив объектов со структурой
+
+// Предположим, что текст не содержит знаков припинания (сплит по пробелам)
+// Делить регуляркками лень
+
+const toObj = function(w) {
+  return { word: w, length: w.length, isCapitalized: w.charAt(0) === w.charAt(0).toUpperCase() }
+}
+
+const toArr = text => text.split(' ').map(toObj)
+
+console.log(toArr('Hello mr reviewer'))
```

Comment:
Попробуйте вызвать функцию со строкой: 
```suggestion
console.log(toArr(`Hey,
mr  student !`))
```

Reply:
В рамках курса задание выполнено корректно, но если вас будут беспокоить недостатки реализации, то можете попробовать исправить их без регулярных выражений, с помощью методов [trim()](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) и [replaceAll()](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) 😉

Reply:
ну без регулярок или с ними, все равно придется перечислять все символы. Иного пути я не нашел, оставлю как есть